src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -104,6 +104,11 @@ export default function NotificationsPane({
               const timeSince = calculatePrettyTimeElapsedSince(
                 createdDate,
               );
+
+              const avatarName =
+                getNotificationProps(notification)[
```

Comment:
`getNotificationProps` has already been run.
This can be changed to:
```
const notificationProps = getNotificationProps(notification)
const {userName, userNameGuid, // everything else being destructured } = notificaitonProps;
//...
const avatarName = notificationProps[currentNotificationSchema?.notificationAvatar] || 'Unnamed User'  // no extra parenthesis needed either
```

Reply:
I didn't know that I could do this. Makes complete sense now. Worldview re-calibrated lol.

Reply:
Good call.